web_action_conditionable/views/view.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<odoo>
-	<template id="assets_backend" name="action conditionable assets" inherit_id="web.assets_backend">
-		<xpath expr="." position="inside">
-			<script type="text/javascript" src="/web_action_conditionable/static/src/js/views.js"></script>
-		</xpath>
-	</template>
-</odoo>
+<openerp>
+    <data>
```

Comment:
When I said "leave it as it was", I was speaking about indentation, not about these deprecated tags...